ospd_openvas/wrapper.py

```diff
@@ -228,14 +248,29 @@ def load_vts(self):
         """ Load the NVT's OIDs and their filename into the vts
         global  dictionary. """
         oids = openvas_db.get_pattern('filename:*:oid')
+        is_custom = 1
         for oid in oids:
             vt_id = oid[1].pop()
-            ret = self.add_vt(vt_id, name=oid[0])
+            filename = oid[0].split(':')
+            ret = self.add_vt(vt_id,
+                              name=filename[1],
+                              custom=nvti.get_nvt_all(vt_id, is_custom))
             if ret == -1:
                 logger.info("Dupplicated VT with OID: {0}".format(vt_id))
             if ret == -2:
                 logger.info("{0}: Invalid OID.".format(vt_id))
 
+    def get_custom_vt_as_xml_str(self, custom):
+        """ Create a string representation of the XML object from the
+        custom data object.
+
+        The custom XML object which is returned will be embedded
```

Comment:
Does the function actually do what its docstring describes? As far as I can tell it merely returns its parameter `custom` as-is.

Also, consider declaring this method a static method as it does not look like it modifies the instance.